### ISSUE TITLE
bug: pnpm create slidev fails to create working project - Slidev Creator

### ISSUE THREAD

**Issue by mfreeman451** [slidevjs/slidev#1106]
**Describe the bug**
Unable to create a working starter project using the prescribed instructions from the official documentation, witih `pnpm`.

**To Reproduce**
1. pnpm create slidev
2. Choose your project name, give it a different name than `slidev`
3. When prompted to Install and start it now? use the default value of `yes`
4. Chose the agent > `pnpm`
5. The program attempts to start but fails because it appears no port has been defined:
```
> slidev --open

node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

RangeError [ERR_SOCKET_BAD_PORT]: options.port should be >= 0 and < 65536. Received 65536.
    at new NodeError (node:internal/errors:371:5)
    at validatePort (node:internal/validators:217:11)
    at Server.listen (node:net:1457:5)
    at /Volumes/T7/src/wtit/eventmanager/eventmanager/node_modules/.pnpm/get-port-please@3.1.0/node_modules/get-port-please/dist/index.cjs:210:12
    at new Promise (<anonymous>)
    at _tryPort (/Volumes/T7/src/wtit/eventmanager/eventmanager/node_modules/.pnpm/get-port-please@3.1.0/node_modules/get-port-please/dist/index.cjs:204:10)
    at checkPort (/Volumes/T7/src/wtit/eventmanager/eventmanager/node_modules/.pnpm/get-port-please@3.1.0/node_modules/get-port-please/dist/index.cjs:353:25)
    at findFreePort (/Volumes/T7/src/wtit/eventmanager/eventmanager/node_modules/.pnpm/@slidev+cli@0.43.0_@types+markdown-it@13.0.1_postcss@8.4.29/node_modules/@slidev/cli/dist/cli.js:499:38)
    at findFreePort (/Volumes/T7/src/wtit/eventmanager/eventmanager/node_modules/.pnpm/@slidev+cli@0.43.0_@types+markdown-it@13.0.1_postcss@8.4.29/node_modules/@slidev/cli/dist/cli.js:501:10)
    at runMicrotasks (<anonymous>) {
  code: 'ERR_SOCKET_BAD_PORT'
}
```

**Desktop (please complete the following information):**
 - OS: osx arm64
 - Browser: N/A
 - Slidev version: Slidev Creator  v0.43.0

**Comment by mfreeman451** [comment]
I think this is just a doc bug.. when I run from the CLI, I can reproduce, if I specify -p <port>, it behaves as expected.

**Comment by antfu** [comment]
I am not able to reproduce your issue. Can you share a reproduction? Thanks

**Comment by antfu** [comment]
We temporarily close this due to the lack of enough information.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)

**Comment by NicklasXYZ** [comment]
I had this problem as well, but with npm. I can share some more info once I get home from work.

**Comment by spydx** [comment]
Had the same problem, using 0.43.0-beta-7.
Tried reproducing using @slidev/cli 0.43.1 but no issues.

The fix was to remove the 0.43.0-beta-7 package and recreate it with the 0.43.1 version instead.
